README.md

```diff
@@ -267,4 +299,4 @@ NVIDIA Tesla V100S     | 14.6 Gh/s
 NVIDIA RTX 4070        | 14.9 Gh/s
 NVIDIA RTX 3080        | 15.2 Gh/s
 NVIDIA RTX 3090        | 17.6 Gh/s
-AMD 7900 XTX           | 27.2 Gh/s
```

Comment:
It's not directly related, but since you're updating the readme already, mind changing this to 23.8?  The user updated it after running for a while and it stabilized.